Content/Tiles/Permafrost/AuroraBrick.cs

```diff
@@ -9,6 +9,11 @@ class AuroraBrick : ModTile
 	{
 		public override string Texture => "StarlightRiver/Assets/Tiles/Permafrost/AuroraBrick";
 
+		public override void Load()
```

Comment:
These changes identical to changes in auroracle-master PR, will need to handle merge accordingly